[2025-09-25T10:00:00.772+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-25T10:00:00.785+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_pipeline_orchestration.check_hdfs_health scheduled__2025-09-25T09:45:00+00:00 [queued]>
[2025-09-25T10:00:00.793+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_pipeline_orchestration.check_hdfs_health scheduled__2025-09-25T09:45:00+00:00 [queued]>
[2025-09-25T10:00:00.793+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-25T10:00:00.803+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_hdfs_health> on 2025-09-25 09:45:00+00:00
[2025-09-25T10:00:00.819+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=1142) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-25T10:00:00.822+0000] {standard_task_runner.py:64} INFO - Started process 1147 to run task
[2025-09-25T10:00:00.816+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_pipeline_orchestration', 'check_hdfs_health', 'scheduled__2025-09-25T09:45:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline_orchestration.py', '--cfg-path', '/tmp/tmp_h68rdq_']
[2025-09-25T10:00:00.823+0000] {standard_task_runner.py:91} INFO - Job 45: Subtask check_hdfs_health
[2025-09-25T10:00:00.896+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_pipeline_orchestration.check_hdfs_health scheduled__2025-09-25T09:45:00+00:00 [running]> on host 791fb6ea4f60
[2025-09-25T10:00:00.975+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_pipeline_orchestration' AIRFLOW_CTX_TASK_ID='check_hdfs_health' AIRFLOW_CTX_EXECUTION_DATE='2025-09-25T09:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-25T09:45:00+00:00'
[2025-09-25T10:00:00.976+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-25T10:00:01.055+0000] {logging_mixin.py:188} INFO - âœ… HDFS NameNode is healthy
[2025-09-25T10:00:01.057+0000] {python.py:237} INFO - Done. Returned value was: HDFS OK
[2025-09-25T10:00:01.057+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-25T10:00:01.083+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_pipeline_orchestration, task_id=check_hdfs_health, run_id=scheduled__2025-09-25T09:45:00+00:00, execution_date=20250925T094500, start_date=20250925T100000, end_date=20250925T100001
[2025-09-25T10:00:01.130+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-25T10:00:01.143+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-25T10:00:01.143+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
